reporter/sources.py

```diff
@@ -355,6 +355,8 @@ class DBQueryErrorsSource(PHPLogsSource):
 * {backtrace}
 """
 
+    ER_LOCK_DEADLOCK = 1213
```

Comment:
`1213` is MySQL's deadlock error code. The name of the constant comes from MySQL source code